docker/hemictl/Dockerfile

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2025 Hemi Labs, Inc.
+# Use of this source code is governed by the MIT License,
+# which can be found in the LICENSE file.
+
+# Build stage
+FROM golang:1.23-alpine3.20@sha256:d0b31558e6b3e4cc59f6011d79905835108c919143ebecc58f35965bf79948f4 AS builder
```

Comment:
Not blocking - The latest available `golang` tag is `1.24.0-alpine3.21` (`sha256:2d40d4fc278dad38be0777d5e2a88a2c6dee51b0b29c97a764fc6c6a11ca893c`), we should probably upgrade all of the base images (although that could be a separate PR): https://hub.docker.com/layers/library/golang/1.24.0-alpine3.21/images/sha256-32ece9b4650da07c5e92744894d2fb9e857729466eeb9e38c99f7ddec83803ec